test/src/test/java/org/corfudb/runtime/clients/LogUnitHandlerTest.java

```diff
@@ -184,11 +187,14 @@ public void quotaExceededTest() throws Exception {
         serverRouter.reset();
         String dirPath = PARAMETERS.TEST_TEMP_DIR;
         final int maxLogSizeInBytes = 1000;
+        FileStore corfuDirBackend = Files.getFileStore(Paths.get(dirPath));
+        final long fsSize = corfuDirBackend.getTotalSpace();
+        final double maxLogSizeInPercentage = maxLogSizeInBytes * 100.0 / fsSize;
         ServerContext sc = serverContext = new ServerContextBuilder()
                 .setMemory(false)
                 .setLogPath(dirPath)
                 .setServerRouter(serverRouter)
-                .setLogSizeQuota(Long.toString(maxLogSizeInBytes))
+                .setLogSizeQuota(Double.toString(maxLogSizeInPercentage))
```

Comment:
The same issue. logSizeQuota -> logSizeLimitPercentage

Reply:
done.